src/script/engine.zig

```diff
@@ -1025,3 +1026,58 @@ test "Script execution - OP_SHA256" {
     Sha256.hash(&[_]u8{1}, &expected_hash, .{});
     try std.testing.expectEqualSlices(u8, expected_hash[0..], hash_bytes);
 }
+
+test "ScriptBuilder Smoke test" {
+    var sb = try ScriptBuilder.new(std.testing.allocator, null);
+    defer sb.deinit();
+
+    var engine = try (try (try (try sb.addInt(1)).addInt(2)).addOpcode(Opcode.OP_ADD)).build();
+
+    try engine.execute();
+
+    defer engine.deinit();
+
+    try std.testing.expectEqual(1, engine.stack.len());
+}
```

Comment:
remove this test and go to scriptBuilder.zig to write actual ScriptBuilder tests.
Where you compare the result of your `.build()` with an expected array of bytes